dist/mlhr-table.js

```diff
@@ -568,11 +571,11 @@ angular.module('datatorrent.mlhrTable.directives.mlhrTable', [
       restrict: 'EA',
       replace: true,
       scope: {
-        columns: '=',
+        _columns: '=columns',
         rows: '=',
         classes: '@tableClass',
         selected: '=',
-        options: '=?',
+        _options: '=?options',
```

Comment:
did you check that this is the correct syntax? As opposed to `_options: '=options?'`? You're probably right, I just dont regularly see the optional syntax with an alternate attr being used...